woocommerce/woocommerce
# Create a helper hook for field validation

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/37004



### How to test the changes in this Pull Request:




1. Pull branch `add/37004`.
2. Run `pnpm run test --filter=@woocommerce/product-editor`.
3. All tests should run successfully.



### Other information:




### FOR PR REVIEWER ONLY:
